### PR TITLE
Js warnings failed prop

### PR DESCRIPTION
**Pull request by icy-meteor**
#### Summary

Failed prop-type error appears in console when agenda's popup is opened.

image

#### Ticket Link
Fixes https://github.com/mattermost/mattermost-plugin-agenda/issues/22

### CODE REVIEW

File: webapp/src/components/meeting_settings/meeting_settings.jsx [hunk 1]
```diff
@@ -153,4 +153,4 @@ export default class MeetingSettingsModal extends React.PureComponent {
             </Modal>
         );
     }
-}
+}
```
Comment (marianunez) [inline webapp/src/components/meeting_settings/meeting_settings.jsx:156]: Nit: New line at the end of file

**Review by marianunez — APPROVED** [review]
Thanks @icy-meteor!

File: webapp/src/components/meeting_settings/meeting_settings.jsx (hunk 1 above)
Comment (icy-meteor) [reply webapp/src/components/meeting_settings/meeting_settings.jsx:156]: It doesn't show up a new line in my code though. Also, if you view this file in Github, it doesn't show the new line. Not sure why it is pointing this edit. 
Github is also showing a small red circle with a minus icon which on hover reads "No newline at end of file".

**Comment by hanzei** [comment]
Tested locally. The JS error doesn't appear any longer. LTGM :+1:

**Pull request merged**